### DISCUSSION TITLE
Github Page is starting zoomed in.

### DISCUSSION THREAD

**Discussion by elliott-evans** [community/community#21741] [Other Feature Feedback, Questions, & Ideas]
<p><a href="https://elliott-evans.github.io/cafe-de-pope/" rel="nofollow noopener">https://elliott-evans.github.io/cafe-de-pope/</a></p>
<p>For some reason, whenever I visit my Github Pages address, it displays the website at a zoomed in size. I have to zoom out to 50% to get the same result as I get from just opening it on the computer.</p>
<p>(Google Chrome)</p>
<p>EDIT: Solved! It was a text sizing issue.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/elliott-evans">@elliott-evans</a>,</p>
<p>Thanks for being here! The site looks okay to me on Chrome. Are you still experiencing issues?</p>

**Comment by elliott-evans** [comment, marked as answer]
<p>Hi there <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> . </p>
<p>I managed to solve the issue on my own. It was a text sizing issue with my code and not Github Pages. </p>
<p>Thank you for the response,</p>
<p>Elliott</p>

**Reply by AlexNelsonMusic** [reply]
Do you mind sharing what exactly the issue was with your code? I'm consistently experiencing this issue so I'm wondering if the way I'm coding text size is bad practice.

**Comment by mekowalski** [comment]
<p>Hello <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>  !</p>
<p>I recently began experiencing this with my Github page.  I do have to zoom down in order to see the whole page at a certain width.  Even when my page is completely mazimized, it’s too zoomed in on Github.  I’m not having this issue with other pages online.  I also noticed this about a week ago.  </p>
<p>What can we do about this?  Any help is appreciated.</p>
<p>-Malind</p>

**Comment by pranjalpagaria** [comment]
I am also facing the same issue . It is automatically  increases in font . Don't know how to solve the problem ? Please help  me .

**Comment by TundracatYT** [comment]
I am facing this issue as well, and the zooming out thing doesn't do much. How do I fix this? btw I am on chromebook and it seems so far as only the clangen github
![Screenshot 2023-09-09 7 48 04 PM](https://github.com/community/community/assets/139393039/ead27196-0915-4b53-8f26-b24d4fabb885)

**Comment by sunny2329** [comment]
I am facing the same issue with all pages. Can anyone help me to fix this .
